### PR TITLE
UWMadison projects

### PR DESCRIPTION
**Pull request by cjabplanalp**

### CODE REVIEW

File: projects/UWMadison_Kwan.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Department: Pharmacy
+Description: ''
```
Comment (brianhlin) [inline projects/UWMadison_Kwan.yaml:2]: Hi @cjabplanalp, is there a good description that you can provide for this project?

File: projects/UWMadison_Ericksen.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Department: Small Molecule Screening Facility
+Description: http://hts.wisc.edu/
```
Comment (brianhlin) [inline projects/UWMadison_Ericksen.yaml:2]: Would it be possible to include some prose here describing the project in addition to the URL?

**Comment by cjabplanalp** [comment]
@brianhlin done!

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**